frontend/public/keycloak.json

```diff
@@ -1,8 +1,8 @@
 {
-  "realm": "xz0xtue5",
-  "auth-server-url": "https://sso-dev.pathfinder.gov.bc.ca/auth",
+  "realm": "quartech",
```

Comment:
I don't think this realm is correct for OpenShift

Reply:
This file is now only used locally, so I switched the config back to the local keycloak config.